.travis.yml

```diff
@@ -4,5 +4,6 @@ rvm:
   - 2.3
   - 2.4
   - 2.5
+  - 2.6
```

Comment:
I'll test it. 

I actually wonder if we should still keep tests for 2.2/3 as they're not maintained anymore.

Reply:
We should probably remove all the EOF version, change it in the gemspec and eventually bump a major version. This could be done eventually in a different PR but it is actually a good point

Reply:
Ok. Let's do it separately. For now I've just removed ruby-head which does not work at travis at the moment (neither 2.7).